src/appleseed/renderer/kernel/lighting/backwardlightsampler.h

```diff
@@ -84,7 +84,7 @@ class BackwardLightSampler
         LightSample&                        light_sample) const;
 
     // Compute the probability density in area measure of a given light sample.
-    float evaluate_pdf(const ShadingPoint& shading_point) const;
+    float evaluate_pdf(const ShadingPoint& shading_point, const ShadingPoint& parent_shading_point) const;
```

Comment:
I think it would be useful to clarify in the comment where are `shading_point` and `parent_shading_point` located (respectively on the light and on the surface being illuminated if I remember correctly? Please check!).

In fact, maybe `shading_point` should be renamed to `light_shading_point`.